zephyr/esp32c3/src/bt/esp_bt_adapter.c

```diff
@@ -1566,4 +1566,28 @@ static void * coex_schm_curr_phase_get_wrapper(void)
 #else
 	return NULL;
 #endif
-}
\ No newline at end of file
+}
+
+esp_power_level_t zephyr_to_esp_ble_power_level(int tx_power) {
```

Comment:
same here and in other esp_bt_adapter.c

Reply:
I will change this approach and ping you back.